man/git-create-branch.md

```diff
@@ -29,11 +39,12 @@ git-create-branch(1) -- Create branches
 
 ## NOTES
 
-As of 4.4.0, the default behavior has changed. `git-create-branch` will no longer automatically setup a remote tracking branch unless the `-r|-remote` option is specified.
+As of 4.4.0, the default behavior has changed. `git-create-branch` will no longer automatically setup a remote tracking branch unless the `-r|-remote` option is specified, however this behavior may be obtained by `remote` preference as described in [Preferences](#PREFERENCES) section.
```

Comment:
Need to point out that the preference support is added in `4.8.0-dev` (we will update the version number when we create the next release).

Reply:
Thanks. I was not sure how to reference version.